Multisig/Logic/Ethereum/SafeContract.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SafeContract.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.02.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class SafeContract: Contract {
```

Comment:
do we need to think about different contracts version for the signing and creating the execTransaction?

Reply:
not before 1.3.0